<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>977</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LV Prüfer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,2,0,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Datenbank:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,1,1">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Verknüpfungen:</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignmentFlag::AlignRight">
           <widget class="QPushButton" name="pushButtonAdd">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignmentFlag::AlignRight">
           <widget class="QPushButton" name="pushButtonDelete">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,3,0">
          <item>
           <widget class="QLabel" name="label_title_left">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Basis:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableView_left"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Langtext:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="textEdit_left"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_save">
            <property name="text">
             <string>Speichern</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,3,0">
          <item>
           <widget class="QLabel" name="label_title_right">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Vergleich:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableView_right"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Langtext:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="textEdit_right"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1293</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProjekt">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionLvImport"/>
    <addaction name="actionProperties"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAnsicht">
    <property name="title">
     <string>Ansicht</string>
    </property>
    <widget class="QMenu" name="menuSortierung_Verkn_pfung">
     <property name="title">
      <string>Sortierung Verknüpfung</string>
     </property>
     <addaction name="actionName"/>
     <addaction name="actionSim"/>
     <addaction name="actionLinked"/>
     <addaction name="separator"/>
     <addaction name="actionAsc"/>
     <addaction name="actionDsc"/>
    </widget>
    <widget class="QMenu" name="menuDifferenz_Bibliothek">
     <property name="title">
      <string>Differenz-Bibliothek</string>
     </property>
     <addaction name="actionAutomatisch"/>
     <addaction name="actionDifflib_2"/>
     <addaction name="actionGoogle_2"/>
    </widget>
    <addaction name="actionSynchron"/>
    <addaction name="menuSortierung_Verkn_pfung"/>
    <addaction name="menuDifferenz_Bibliothek"/>
    <addaction name="actionAlles_ausklappen"/>
   </widget>
   <widget class="QMenu" name="menuBearbeiten">
    <property name="title">
     <string>Bearbeiten</string>
    </property>
    <addaction name="actionSuchen"/>
    <addaction name="actionSuchen_Alle"/>
   </widget>
   <addaction name="menuProjekt"/>
   <addaction name="menuBearbeiten"/>
   <addaction name="menuAnsicht"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Ordner öffnen</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionLinksCreate">
   <property name="text">
    <string>Erstellen</string>
   </property>
  </action>
  <action name="actionLinksLoad">
   <property name="text">
    <string>Laden</string>
   </property>
  </action>
  <action name="actionLinksSave">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="actionDifflib">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>difflibparser (schnell)</string>
   </property>
  </action>
  <action name="actionGoogle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>google (präzise)</string>
   </property>
  </action>
  <action name="actionSynchron">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Synchron scrollen</string>
   </property>
  </action>
  <action name="actionName">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </action>
  <action name="actionSim">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ähnlichkeit</string>
   </property>
  </action>
  <action name="actionLinked">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Verknüpft</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Eigenschaften</string>
   </property>
  </action>
  <action name="actionGoogle_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>google (präzise)</string>
   </property>
  </action>
  <action name="actionDifflib_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>difflibparser (schnell)</string>
   </property>
  </action>
  <action name="actionLv">
   <property name="text">
    <string>1. Erzeuge Übersicht</string>
   </property>
  </action>
  <action name="actionLv_plus">
   <property name="text">
    <string>2. Importiere LV</string>
   </property>
  </action>
  <action name="actionLvImport">
   <property name="text">
    <string>LV importieren</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAsc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Aufsteigend</string>
   </property>
  </action>
  <action name="actionDsc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Absteigend</string>
   </property>
  </action>
  <action name="actionAutomatisch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>automatisch</string>
   </property>
  </action>
  <action name="actionAlles_ausklappen">
   <property name="text">
    <string>Alles ausklappen</string>
   </property>
  </action>
  <action name="actionSuchen">
   <property name="text">
    <string>Suchen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSuchen_Alle">
   <property name="text">
    <string>Suchen Alle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
